packages/trigger-pager/src/triggerPager.test.ts

```diff
@@ -1,38 +1,97 @@
+  const actualPath = require.resolve(specifier)
+  if (arguments.length === 1) {
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    require.cache[actualPath] = require(`../__mocks__/${specifier}`)
+  } else {
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    const actual = require(specifier)
+    const Module = require('node:module') // eslint-disable-line global-require
+    require.cache[actualPath] = new Module(actualPath, module)
+    Object.defineProperties(require.cache[actualPath], {
+      exports: {
+        // @ts-expect-error -  Object literal may only specify known properties, and '__proto__' does not exist in type 'PropertyDescriptor'
+        __proto__: null,
+        value: replacer(actual),
+      },
+      // @ts-expect-error -  Object literal may only specify known properties, and '__proto__' does not exist in type 'PropertyDescriptor'
+      resetFn: { __proto__: null, value: replacer.bind(null, actual) },
+    })
+  }
+}
+
+// eslint-disable-next-line no-unused-vars
+const mockPost = mock.fn(async (endpoint: string, payload: unknown) => {
+  throw Error('mock post for each test')
+})
+
+mockRequire('@pagerduty/pdjs', () => {
+  return { api: () => ({ post: mockPost }) }
+})
+
+const { default: triggerPager } = require('./triggerPager')
+
+const LOREM_LONG = `Lorem ipsum dolor sit amet, consectetur adipiscing elit,
 sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
 Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
 nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
 reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
 pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
 culpa qui officia deserunt mollit anim id est laborum.`
 
-const LOREM3 = `${LOREM} ${LOREM} ${LOREM}`
-
-// const api = mocked(apiX)
-jest.mock('@pagerduty/pdjs')
+const LOREM_SHORT = `Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.`
 
 describe('triggerPager', () => {
   test('main', async () => {
-    const post = jest.fn(async (endpoint: $TSFixMe, payload: $TSFixMe) => {
-      expect(endpoint).toBe('/incidents')
-      expect(payload).toMatchSnapshot()
+    // eslint-disable-next-line no-unused-vars
+    mockPost.mock.mockImplementationOnce(async (endpoint: string, payload: unknown) => {
       return { data: { error: null } }
     })
-    // @ts-expect-error
-    api.mockReturnValue({ post })
+
+    const serviceId = crypto.randomUUID()
+
     await triggerPager({
-      title: LOREM3,
-      description: LOREM3,
-      serviceId: 'SERVICE_ID',
+      title: LOREM_SHORT,
+      description: LOREM_LONG,
+      serviceId,
+    })
+
+    assert.equal(mockPost.mock.callCount(), 1)
+    assert.equal(mockPost.mock.calls[0].arguments[0], '/incidents')
+    assert.deepEqual(mockPost.mock.calls[0].arguments[1], {
```

Comment:
```suggestion
    assert.strictEqual(mockPost.mock.callCount(), 1)
    assert.strictEqual(mockPost.mock.calls[0].arguments[0], '/incidents')
    assert.deepStrictEqual(mockPost.mock.calls[0].arguments[1], {
```
note it's also possible to import `node:assert/strict`, but IMO being explicit like in my suggestion above is clearer